configuration/configuration_test.go

```diff
@@ -111,47 +112,13 @@ func TestBlobCorrectPayload(t *testing.T) {
 }
 
 func equalPayload(p1, p2 payloads.Configure) bool {
-	return (p1.Configure.Scheduler.ConfigStorageURI == p2.Configure.Scheduler.ConfigStorageURI &&
-
-		p1.Configure.Controller.VolumePort == p2.Configure.Controller.VolumePort &&
-		p1.Configure.Controller.ComputePort == p2.Configure.Controller.ComputePort &&
-		p1.Configure.Controller.HTTPSCACert == p2.Configure.Controller.HTTPSCACert &&
-		p1.Configure.Controller.HTTPSKey == p2.Configure.Controller.HTTPSKey &&
-		p1.Configure.Controller.IdentityUser == p2.Configure.Controller.IdentityUser &&
-		p1.Configure.Controller.IdentityPassword == p2.Configure.Controller.IdentityPassword &&
-
-		libsnnet.EqualNetSlice(p1.Configure.Launcher.ComputeNetwork, p2.Configure.Launcher.ComputeNetwork) &&
-		libsnnet.EqualNetSlice(p1.Configure.Launcher.ManagementNetwork, p2.Configure.Launcher.ManagementNetwork) &&
-		p1.Configure.Launcher.DiskLimit == p2.Configure.Launcher.DiskLimit &&
-		p1.Configure.Launcher.MemoryLimit == p2.Configure.Launcher.MemoryLimit &&
-
-		p1.Configure.ImageService.Type == p2.Configure.ImageService.Type &&
-		p1.Configure.ImageService.URL == p2.Configure.ImageService.URL &&
-
-		p1.Configure.IdentityService.Type == p2.Configure.IdentityService.Type &&
-		p1.Configure.IdentityService.URL == p2.Configure.IdentityService.URL)
+	return reflect.DeepEqual(p1, p2)
```

Comment:
@rbradford it could - it'd require that I force push to update the patch though - which I can do if people would rather I do that and squash in the updates I made for mark's feedback.